python/pyspark/sql/functions.py

```diff
@@ -1779,7 +1779,7 @@ def month(col):
 
 def dayofweek(col):
     """
-    Extract the day of the week of a given date as integer.
+    Extract the day of the week of a given date as integer with Sunday=1, Saturday=7.
```

Comment:
can you copy the docs in `dayofweek` at `functions.scala`? it would also be great to update `functions.R` docs together.

Reply:
Thanks for the comment. I took the exact wording from functions.scala and also updated functions.R